### PR TITLE
Add notes about shortmess options to doc and readme files. (#59)

### PR DESCRIPTION
**Pull request by pdavydov108**
See issue #59 for explanation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -202,6 +202,17 @@ As a source author you do not have to worry about synchronization issues in case
 characters. asyncomplete.vim uses partial caching as well as ignores if the context changes when calling `asyncomplete#complete`.
 This is one of the core reason why the original context must be passed when calling `asyncomplete#complete`.
 
+### Suppress completion menu messages
```
Comment (prabirshrestha) [inline README.md]: No need to add it in read me. I used it because we didn’t have docs yet.

File: doc/asyncomplete.txt [hunk 2]
```diff
@@ -5,9 +5,9 @@
 ===============================================================================
 CONTENTS                                                *asyncomplete-contents*
 
-    1. Introduction        |asyncomplete-introduction|
-    2. Options             |asyncomplete-options|
-
+    1. Introduction              |asyncomplete-introduction|
```
Comment (prabirshrestha) [inline doc/asyncomplete.txt]: should use spaces instead of tabs.

File: README.md (hunk 1 above)
Comment (pdavydov108) [reply README.md]: Are you sure about that? I think for now people look into README more then to doc, cause doc for this plugin doesn't have too much info.

File: doc/asyncomplete.txt (hunk 2 above)
Comment (pdavydov108) [reply doc/asyncomplete.txt]: Ups, sorry, I'll fix it! Looks like some plugin kindly inserted tabs for me, I personally don't use them...

File: README.md (hunk 1 above)
Comment (prabirshrestha) [reply README.md]: @pdavydov108 I would want to have docs in README as possible.
I need to remove the ones that are in there and move most of them to doc.

File: doc/asyncomplete.txt (hunk 2 above)
Comment (pdavydov108) [reply doc/asyncomplete.txt]: Hm, I've checked the doc and haven't found any tabs there. Are you sure they are there? I use `expandtab` in my vimrc, so I think even if I press `tab` it inserts four spaces automatically. I tried to highlight them with `set list && set listchars=tab:>-` but it showed me nothing (I checked that this command works for me if I force vim to insert tab via ctrl-v + tab).

Comment (prabirshrestha) [reply doc/asyncomplete.txt]: I will fix this once merged.

**Pull request merged**